### PR TITLE
Personal portfolio review

### PR DESCRIPTION
**Pull request by Yerimah**
## Trial Review Project

This branch contains the code to be reviewed

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+.projects-grid {
+  display: flex;
+  row-gap: 50px;
+  width: 100vw;
+  min-height: 50vh;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  align-items: center;
+}
```
Comment (Yerimah) [inline style.css:97]: Good job on implementing flexbox. I think there is something wrong with the display of your projects in the project section. In order to ensure your projects are properly displayed, you should use a CSS grid in place of flex

File: style.css [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+  background-image: linear-gradient(62deg, #3a3d40 0%, #181719 100%);
+}
+
+#navbar {
+  display: flex;
+  width: 100vw;
+  justify-content: flex-end;
+  position: fixed;
+  background-color: #be3144;
+}
```
Comment (Yerimah) [inline style.css:25]: originalcopy (2)

portfolio nav

- I kindly suggest you make your design conform to the original design as much as possible. Include the hamburger menu on the left side of the navbar in your design.

**Review by Yerimah — COMMENTED** [review]
# STATUS: CHANGES REQUIRED

Hi!
Good Job so far! 👍 
There are some issues you still need to work on to progress to the next project but you are almost done.

### Project highlights:
 - All linter checks are passing ✔️
 - Your README file looks very professional and the PR has a descriptive title/summary of the implemented changes 📝
 - Descriptive commits and correct use of the GitHub Flow 🔥

## Required Changes ♻️
*Check the comments under the review.*

## Optional suggestions
*Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.*



Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

***Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.***

**Review by MrBrN197 — APPROVED** [review]
## Hi, @Yerimah 👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**